### PR TITLE
extra symbols

### PR DESCRIPTION
**Pull request by bdeket**
Some symbols I miss from time to time:
double/tripple integrals
double struck uppercase letters
angle

### CODE REVIEW

**Comment by rfindler** [comment]
Each time we add something to this list, there is a possibility that the DrRacket manual build breaks because scribble runs into new unicode characters that it isn't translating into latex properly. Probably the simplest way to find those breakages is to do this:

```
$ cd ...../drracket/drracket/scribblings/drracket
$ scribble --dest x --latex drracket.scrbl && (cd x ; pdflatex drracket)
```

where the five ellipses are the place where the drracket package is installed.

Doing that with the commit in this pull request in place results in this at the end of the latex output:

```
See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.3514 \hbox{\mbox{\hphantom{\Scribtexttt{x}}}∠
                                                 } \\
? x
```

which suggest that [this function](https://github.com/racket/scribble/blob/master/scribble-lib/scribble/latex-render.rkt#L1296) needs another line.

I'll take a look into adding these -- I just tried and `$\\angle` seems to work for the first one. (I'm out of time for the moment but I can return to this later on today, I expect.)

Another thing we might consider doing is changing the DrRacket manual but I lean towards not doing that.

**Comment by rfindler** [comment]
This [wikipedia page](https://en.wikipedia.org/wiki/Blackboard_bold) looks helpful for finding the actual characters.

**Comment by bdeket** [comment]
Thank you for looking into this and sorry for not following up.
Can I help out with completing this?
Do I understand correctly that `Naturals` etc. should be changed `\bN` etc.

**Comment by rfindler** [comment]
I didn't merge the PR yet (but I'm happy to do so if you want!). I actually lost track of it! But the changes to Scribble are done, I think.

We can have multiple names for a given symbol so if you use a latex package that uses `\Naturals`, etc, then we can use that name. We can also use `\bN` and for any of the symbols listed in the "Unicode symbol" column of [this table on the black-board bold wikipedia page](https://en.wikipedia.org/wiki/Blackboard_bold#Usage); at least, I think those are all covered by the commit I did to scribble above.

How would you like to proceed?

File: tex-table/tex-table.rkt [hunk 1]
```diff
@@ -190,14 +191,26 @@
...
+    ("oiiint" "∰")
 
     ("sqrt" "√")
     
     ("skull" "☠") 
     ("smiley" "☺")
     ("blacksmiley" "☻")
     ("frownie" "☹")
+
+    ("Naturals" "ℕ")
```
Comment (sorawee) [inline tex-table/tex-table.rkt:207]: Should it be `Naturals` or `Natural`? If former, shouldn't `Complex` below be `Complexes` for consistency?

**Comment by rfindler** [comment]
My take on this is that we should use "\Naturals" if that's already
established in some fairly commonly used latex package. Ditto any other
names.

Robby

**Comment by bdeket** [comment]
The standard seems to be to use the `\mathbb` approach.
I've found this list [https://www.rollpie.com/post/533](https://www.rollpie.com/post/533) which seem to come from the `texvc` package. But this lacks uniformity and misses Rationals.

**Comment by sorawee** [comment]
I would vote to not follow `texvc`. It's not commonly used.

**Comment by sorawee** [comment]
Wait, it looks like @rfindler already pushed the initial commit to the master branch last month.

https://github.com/racket/gui/blob/master/tex-table/tex-table.rkt#L207

**Comment by sorawee** [comment]
@rfindler what do you think should be the next step? Do you think we should just cherry-pick 2d23378d7669faecc96a4f43c476e5dbc61eacde and then close the PR?

**Comment by rfindler** [comment]
I can add the other \b<letter> shortcuts; they seem useful to me. Shall I do that? Maybe after cherry-picking?

**Comment by sorawee** [comment]
I'd be OK with that.

**Comment by rfindler** [comment]
Thanks for this @bdeket ! I've cherry-picked the commit.

**Comment by shhyou** [comment]
Late to the party but +1 for the currently just-pushed `\bN`, etc. Other same abbrvs:
https://github.com/agda/agda/blob/ee876ee229cdc2f3b1ff80f6d60568a1869f9385/src/data/emacs-mode/agda-input.el#L516

**Comment by rfindler** [comment]
It looks like the lowercase bb letters don't work with the current set of latex includes we use. That is, the `\mathbb` macro doesn't like them. [This stackexchange answer](https://tex.stackexchange.com/questions/254445/package-for-mathbblowercase-letter-symbol) suggests that we could add a new dependency on the latex generated output but that seems scary. So, for now, I think I'll just do the uppercase letters and the digits.

**Comment by rfindler** [comment]
.... actually it doesn't work for the digits either! So I'll leave those out for now too. sigh.

**Comment by rfindler** [comment]
Well, I just added them and just wrote an English explanation in the DrRacket manual. FWIW.

**Comment by bdeket** [comment]
Thank you very much.

**Pull request closed without merging**